ultravox/inference/infer.py

```diff
@@ -11,7 +11,7 @@
 from ultravox.model import ultravox_processing
 
 SAMPLE_RATE = 16000
-MAX_TOKENS = 1024
+MAX_NEW_TOKENS = 1024
```

Comment:
Change `MAX_TOKENS` to `MAX_NEW_TOKENS` to be consistent with the parameter name `max_new_tokens` in `self.model.generate()`